setup_cfg_fmt.py

```diff
@@ -398,7 +398,9 @@ def format_file(
     # set license fields if a license exists
     license_filename = _first_file(filename, 'licen[sc]e')
     if license_filename is not None:
-        cfg['metadata']['license_file'] = os.path.basename(license_filename)
+        license_basename = os.path.basename(license_filename)
+        if license_basename not in cfg['metadata'].get('license_files', ''):
```

Comment:
this should be a little more strict -- string containment isn't precise enough of a check

Reply:
ah I see your other message out of order -- the rest of them use `.strip().splitlines()`